infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationSinkManager.java

```diff
@@ -481,6 +475,11 @@ private synchronized void startSnapshotApplyAsync(LogReplication.LogReplicationE
 
     private synchronized void startSnapshotApply(LogReplication.LogReplicationEntryMsg entry) {
         log.debug("Entry Start Snapshot Sync Apply, id={}", entry.getMetadata().getSyncRequestId());
+        //set data_consistent as false and clear streams which has data to be replicated and the streams
```

Comment:
nit: should this be "which has data to be applied" or "which has already data replicated"?

Reply:
yup, changed it to "data evidenced from active"